libcaf_core/caf/flow/step/on_error_return_item.hpp

```diff
@@ -0,0 +1,47 @@
+// This file is part of CAF, the C++ Actor Framework. See the file LICENSE in
+// the main distribution directory for license terms and copyright or visit
+// https://github.com/actor-framework/actor-framework/blob/master/LICENSE.
+
+#pragma once
+
+#include "caf/detail/type_traits.hpp"
+#include "caf/fwd.hpp"
+
+#include <utility>
+
+namespace caf::flow::step {
+
+template <class T>
+class on_error_return_item {
+public:
+  using input_type = T;
+
+  using output_type = T;
+
+  explicit on_error_return_item(T&& item) : item_(std::forward<T>(item)) {
```

Comment:
This is an rvalue ref, calling `forward` on it makes little sense. The common pattern is pass by value and then move into place.

```suggestion
  explicit on_error_return_item(T item) : item_(std::move(item)) {
```